textlint-rule-preset-techpit-host-guide/src/codeblock-modify-comment.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @param {RuleContext} context
+ */
+module.exports = function(context, options = {}) {
+  const { Syntax, RuleError, report, getSource } = context;
+  return {
+    [Syntax.CodeBlock](node) {
+      const text = getSource(node);
+      const words = ["追加", "削除", "変更"];
```

Comment:
```suggestion
      const words = ["追加", "削除", "変更", "編集"];
```
これらに加えて過去に「編集」くらいのパターンは見たような気がするので…